fr-fr/javascript-fr.html.markdown

```diff
@@ -1,12 +1,12 @@
 ---
 language: javascript
 contributors:
-    - ['Leigh Brenecki', 'https://leigh.net.au']
```

Comment:
changing the quotes seems to be an unrelated change that doesn't improve the code

Reply:
it conforms with the specs on [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown#header-configuration)

(truth: my formatter did it)

Reply:
okay, then we can use double quotes